README.md

```diff
@@ -94,9 +94,9 @@ If you're using a certbot version below `v1.7` all options related to external p
     --certbot-dns-godaddy:dns-godaddy-credentials
     --certbot-dns-godaddy:dns-godaddy-propagation-seconds
 
-Further, every occurence of `dns_godaddy` in the config file must be prefixed by `certbot_dns_godaddy:`, resulting in a file like this:
+Same apploes to the config file, every occurence of `dns_godaddy` in file, must be prefixed by `certbot_dns_godaddy:` resulting in a file looking like this:
```

Comment:
applies

Reply:
Fixed

Reply:
Still says apploes

Reply:
In the branch for this PR, this is fixed. I'm not sure why you're seeing that it is not.